src/containers/ConnectionManagement.jsx

```diff
@@ -57,6 +63,13 @@ class ConnectionManagement extends Component {
     }
 
     this.store.fetchDriveUrl()
+
+    if (!this.props.existingAccount) {
+      if (this.props.isCreating) {
+        throw new Error('Connection creation already in progress')
```

Comment:
What happens if you `throw` in a component `constructor`? Doesn't stuff just *break* without a way to catch it?

Reply:
You're probably right, instead I will just console.warn something and prevent the `isCreating` to be "fired" twice.